nibabel/gifti/gifti.py

```diff
@@ -104,23 +119,38 @@ def print_summary(self):
 
 
 class GiftiLabel(xml.XmlSerializable):
-    """
-    key = int
-    label = str
-    # rgba
-    # freesurfer examples seem not to conform
-    # to datatype "NIFTI_TYPE_RGBA32" because they
-    # are floats, not unsigned 32-bit integers
-    red = float
-    green = float
-    blue = float
-    alpha = float
+    """ Gifti label: association of integer key with optional RGBA values
+
+    Quotes are from the gifti spec dated 2011-01-14.
+
+    Attributes
+    ----------
+    key : int
+        (From the spec): "This required attribute contains a non-negative
+        integer value. If a DataArray's Intent is NIFTI_INTENT_LABEL and a
+        value in the DataArray is 'X', its corresponding label is the label
+        with the Key attribute containing the value 'X'. In early versions of
+        the GIFTI file format, the attribute Index was used instead of Key. If
+        an Index attribute is encountered, it should be processed like the Key
+        attribute."
+    red : None or float
+        Optional value for red.
+    green : None or float
+        Optional value for green.
+    blue : None or float
+        Optional value for blue.
+    alpha : None or float
+        Optional value for alpha.
+
+    Notes
+    -----
+    freesurfer examples seem not to conform to datatype "NIFTI_TYPE_RGBA32"
+    because they are floats, not 4 8-bit integers.
     """
 
     def __init__(self, key=0, label='', red=None, green=None, blue=None,
                  alpha=None):
         self.key = key
-        self.label = label
```

Comment:
Guessing you're removing this because `GiftiLabel().label` isn't referenced anywhere? If so, should it still be in the parameter list?